frontend/src/components/App.js

```diff
@@ -42,8 +40,8 @@ class App extends Component {
         try {
             const web3 = await getWeb3();
             this.props.initializeWeb3(web3);
-            // TODO: Bootstrap the operator url during build
-            const salad = new CoinjoinClient( undefined, web3 );
+            const protocolHost = window.location.protocol + '//' + window.location.hostname;
```

Comment:
We should define an endpoint for the operator api (e.g. http://host/api). The frontend should be in the domain root. A reverse proxy should rewrite the api route to the operator internal endpoint.

Reply:
I don't really know how to set up and configure nginx. do you think you could do it as a PR to this branch? (or, just a commit on this branch?)

Reply:
I think that it can be a separate PR. We just need to keep track of it. I can take care of it.